add xo

j-em opened the pull request:
This package add support for [XO](https://github.com/sindresorhus/xo) (a JS linter) in compile mode.
I am the maintainer. Building and installation seem to work fine on my machine.

Link to repo: https://github.com/j-em/xo-emacs

Let me know if something should be changed :-)

purcell commented:
Thanks. Quick feedback:

- "emacs" in the package description line is redundant
- `setup-xo` should be `xo-setup`
- Obviously the user is meant to call `xo-setup`, so consider putting that info in the `Commentary:` section of the comments: it's not helpful to only put info in the README if people will be installing this via the package. :-)

Hope that helps!

j-em commented:
Thanks for the feedback !

- Removed emacs from the description line
- Renamed `setup-xo` to `xo-setup`
- Added something in `Commentary` about the need to call `xo-setup`

Let me know if there's anything else :)

purcell commented:
Super, thanks.
